src/test/java/com/apache/hadoop/hive/ql/parse/ParseDriverTest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.apache.hadoop.hive.ql.parse;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.apache.hadoop.hive.ql.parse.ASTNode;
+import org.apache.hadoop.hive.ql.parse.MutantSwarmParseDriver;
+import org.apache.hadoop.hive.ql.parse.ParseException;
+import org.junit.Test;
+
+public class ParseDriverTest {
+
+  private ASTNode node;
+
+  private String command = "CREATE TABLE foobar AS\nSELECT c\nFROM bar\nWHERE b = 3";
+  private String command2 = "CREAGGGTE TABLE foobar AS\nSELECT c ERROR \nFROM bar\nWHERE b = 3";
```

Comment:
```suggestion
  private String invalidCommand = "CREAGGGTE TABLE foobar AS\nSELECT c ERROR \nFROM bar\nWHERE b = 3";
```
so the name clearly says what it is. Also, since this isn't used outside of the method you might as well move it to method-level scope instead of class-level. Same for `command` above. We always aim to keep the scope of members as limited as possible.